dev-docs/source/conf.py

```diff
@@ -16,7 +16,7 @@
     def setup(app):
         """Called automatically by Sphinx when starting the build process
         """
-        app.add_stylesheet("custom.css")
+        app.add_css_file("custom.css")
```

Comment:
This doesn't work on non-conda builds because the Sphinx version is old.

Reply:
Multiple version support added